.github/workflows/vast.yaml

```diff
@@ -1079,23 +607,448 @@ jobs:
+      CTEST_OUTPUT_ON_FAILURE: YES
+      DEBIAN_FRONTEND: noninteractive
+      HOMEBREW_GITHUB_API_TOKEN: ${{ github.token }}
+      HOMEBREW_NO_ANALYTICS: 1
+      HOMEBREW_NO_INSTALL_CLEANUP: 1
+      HOMEBREW_NO_AUTO_UPDATE: 1
+      DESTDIR: "${{ github.workspace }}"
+    name: ${{ matrix.plugin.name }} Plugin
+    steps:
+      - name: Setup ENV
+        run: |
+           if ${{ matrix.plugin.os == 'ubuntu-20.04' }}; then
+            CC=gcc-10
+            CXX=g++-10
+            PACKAGE_SUFFIX=Release-GCC
+           elif ${{ matrix.plugin.os == 'macos-latest' }}; then
+            CC=clang
+            CXX=clang++
+            PACKAGE_SUFFIX=Release-Clang
+           fi
+           echo "CC=${CC}" >> $GITHUB_ENV
+           echo "CXX=${CXX}" >> $GITHUB_ENV
+           echo "PACKAGE_SUFFIX=${PACKAGE_SUFFIX}" >> $GITHUB_ENV
+      - name: Install git for debian
+        if: ${{ matrix.plugin.os == 'ubuntu-20.04' }}
+        run: |
+          apt-get update
+          apt-get -y install git
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Install Dependencies macos
+        if: ${{ matrix.plugin.os == 'macos-latest' }}
+        run: |
+          brew install \
+            ${{ join(matrix.plugin.dependencies, ' ') }} \
+            curl
+          ./scripts/macOS/install-dev-dependencies.sh
+          python3 -m pip install --upgrade pip
+      - name: Install Dependencies ubuntu
+        if: ${{ matrix.plugin.os == 'ubuntu-20.04' }}
+        run: |
+          apt-get -y install \
+            ${{ join(matrix.plugin.dependencies, ' ') }} \
+            apt-transport-https \
+            curl
+          ./scripts/debian/install-dev-dependencies.sh
+          python3 -m pip install --upgrade pip
+      - name: Determine VAST Package Name
+        id: configure
+        run: |
+          build_version=${{ needs.configure.outputs.build-version }}
+          if ${{ needs.configure.outputs.run-vast == 'false' }}; then
+            build_version=${{ needs.configure.outputs.before-version }}
+          fi
+          PACKAGE_NAME="$(echo "vast-${build_version}-$(uname -s)-${PACKAGE_SUFFIX}" | awk '{ print tolower($0) }')"
+          echo "PACKAGE_NAME=$PACKAGE_NAME" >> $GITHUB_ENV
+      - name: Download VAST artifact
+        if: ${{ needs.configure.outputs.run-vast == 'true' }}
+        uses: actions/download-artifact@v3
+        with:
+          name: "${{ env.PACKAGE_NAME }}.tar.gz"
+      - name: Configure GCS Credentials
+        if: ${{ needs.configure.outputs.run-vast == 'false' }}
+        uses: google-github-actions/setup-gcloud@v0
+        with:
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
+          export_default_credentials: true
+      - name: Download VAST
+        if: ${{ needs.configure.outputs.run-vast == 'false' }}
+        run: |
+          gsutil cp "gs://${{ secrets.GCS_BUCKET }}/${{ env.PACKAGE_NAME }}.tar.gz" .
```

Comment:
How do we test this? Just by merging and opening another PR that only modifies a plugin?

Reply:
I have created a new branch from this one. Then i have created PR and set the merge target to current PR's branch.
I have changed something in one plugin to trigger changes in /plugins. I have also changed some SHAs but i am not sure if it was to affect this job. It was working under such circumstances

Reply:
Okay, that should be good enough to have some confidence. 👍